int8 WHEEL = 0
int8 SPIGOT = 1
int8 SHUTTLECOCK = 2
int8 BREAKERA = 3
int8 BREAKERB = 4
int8 NONE = 5

int8 device_type
---
int8 no_response
